src/xb-tool.c

```diff
@@ -456,6 +481,8 @@ main (int argc, char *argv[])
 			"Return only when the silo is no longer valid", NULL },
 		{ "profile", 'p', 0, G_OPTION_ARG_NONE, &priv->profile,
 			"Show profiling information", NULL },
+		{ "tokenize", 'p', 0, G_OPTION_ARG_STRING_ARRAY, &priv->tokenize,
```

Comment:
You can’t re-use the same `'p'` short name for the option here; it’s already used by `--profile`. Might also help to give an example of what values to provide to `--tokenize`.

Reply:
Ahh, typo, fixed. I've also added an example, thanks.